src/Framework/QueryBuilder/Concerns/WhereClause.php

```diff
@@ -487,7 +487,7 @@ private function buildWhereSQL(Where $where)
                     $where->comparisonOperator,
                     strpos($where->value, '%') !== false
                         ? $where->value
-                        : sprintf('%%%s%%', $where->value )
+                        : '%' . $where->value . '%'
```

Comment:
@kjohnson So is this broken, too? It looks like it's doing the double %% correctly?

Reply:
The double `%%` is only needed when used in the first argument of the `WPDB::prepare()` method. In this case, the wildcard is in the "complete LIKE string" since it is passed as a substitution argument.